### PR TITLE
Add error animation for feedback view for vivo

### PR DESCRIPTION
**Pull request by Gomay88**
Ticket -> https://jira.tid.es/browse/IOS-7087

**WHAT**
Need to add the error animation when displaying feedback error view

**HOW**
Added Movistar animation to vivo images and try the changes in MisticaCatalog

vivo error

### CODE REVIEW

**Comment by yceballost** [comment]
A designer as a reviewer is missing in this PR!!!!! :___(

**Comment by DavidMarinCalleja** [comment]
> A designer as a reviewer is missing in this PR!!!!! :___(

you are right. @Gomay88  add to @yceballost  to this pr, please

**Comment by Gomay88** [comment]
> > A designer as a reviewer is missing in this PR!!!!! :___(
> 
> you are right. @Gomay88 add to @yceballost to this pr, please

I can't add him, when I search him in reviewers he doesn't appear

**Review by jmpg93 — APPROVED** [review]

File: Mistica/Source/Resources/Assets.xcassets/vivo/FeedbackError.dataset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (yceballost) [inline Mistica/Source/Resources/Assets.xcassets/vivo/FeedbackError.dataset/Contents.json:1]: Can you generate a version instead a video to review? I can't confirm if haptic feedback is working with the video, but If you can confirm this, I can approve this PR!

**Review by yceballost — CHANGES_REQUESTED** [review]

**Review by alexanegon — APPROVED** [review]

File: Mistica/Source/Resources/Assets.xcassets/vivo/FeedbackError.dataset/Contents.json (hunk 1 above)
Comment (Gomay88) [reply Mistica/Source/Resources/Assets.xcassets/vivo/FeedbackError.dataset/Contents.json:1]: You can try with the one I just generate. is the last one of Mistica catalog

**Review by DavidMarinCalleja — APPROVED** [review]
Good job!

File: Mistica/Source/Resources/Assets.xcassets/vivo/FeedbackError.dataset/Contents.json (hunk 1 above)
Comment (yceballost) [reply Mistica/Source/Resources/Assets.xcassets/vivo/FeedbackError.dataset/Contents.json:1]: works fine!

**Review by yceballost — APPROVED** [review]
gj! thanks!

**Pull request merged**